### PR TITLE
Bugfix/evaluation fixes

### PR DESCRIPTION
**Pull request by sebbader**
Tested the ontology with TripleChecker and Oops and corrected the severe issues.

### CODE REVIEW

File: taxonomies/Message.ttl [hunk 1]
```diff
@@ -162,7 +162,7 @@ ids:ParticipantUpdateMessage a owl:Class;
 ids:revokeReason
     a owl:DatatypeProperty;
     rdfs:label "revoke reason"@en;
-    rdfs:domain ids:ConnectorCertificateRevokedMessage, ids:ParticipantCertificateRevokedMessage ;
+    rdfs:domain ids:ConnectorCertificateRevokedMessage; # TODO: , ids:ParticipantCertificateRevokedMessage ;
```
Comment (IDS-Travis) [inline taxonomies/Message.ttl]: Why the change?

Comment (HaydarAk) [inline taxonomies/Message.ttl]: Why change ?
